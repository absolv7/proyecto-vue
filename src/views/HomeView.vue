<template>
	<div class="home">
		<b-button block v-b-toggle.accordion-1 variant="primary">
			Info
		</b-button>

		<b-collapse id="accordion-1" accordion="my-accordion" role="tabpanel">
			<b-card-body>
				Lorem Ipsum is simply dummy text of the printing and typesetting
				industry. Lorem Ipsum has been the industry's standard dummy
				text ever since the 1500s, when an unknown printer took a galley
				of type and scrambled it to make a type specimen book. It has
				survived not only five centuries, but also the leap into
				electronic typesetting, remaining essentially unchanged. It was
				popularised in the 1960s with the release of Letraset sheets
				containing Lorem Ipsum passages, and more recently with desktop
				publishing software like Aldus PageMaker including versions of
				Lorem Ipsum.
			</b-card-body>
		</b-collapse>

		<h3></h3>
	</div>
</template>

<script>
export default {
	name: 'HomeView',
};
</script>
