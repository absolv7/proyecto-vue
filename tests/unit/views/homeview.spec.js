import { shallowMount } from "@vue/test-utils";
import HomeView from "@/views/HomeView";


describe("Componente HomeView", () => {
  test("Debe hacer match con el snapshot", () => {

    // Monto el componente HomeView
    const wrapper = shallowMount(HomeView);

    //Espero que el html sea identico al snapshot.
    expect(wrapper.html()).toMatchSnapshot();
  });
});
