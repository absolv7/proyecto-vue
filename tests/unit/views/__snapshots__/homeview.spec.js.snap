// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente HomeView Debe hacer match con el snapshot 1`] = `
<div class="home">
  <h1>PROSEGUR<br>SOLUTIONS</h1>
  <hr>
  <div role="tablist" class="accordion">
    <b-card no-body="" class="mb-3">
      <b-card-header header-tag="header" role="tab" class="p-1">
        <b-button block="" variant="warning">Alarms</b-button>
      </b-card-header>
      <b-collapse id="accordion-1" visible="" accordion="my-accordion" role="tabpanel">
        <b-card-body>
          <b-card-text>Offers high quality solutions for residential and commercial alarms.</b-card-text>
        </b-card-body>
      </b-collapse>
    </b-card>
    <b-card no-body="" class="mb-3">
      <b-card-header header-tag="header" role="tab" class="p-1">
        <b-button block="" variant="warning">Security</b-button>
      </b-card-header>
      <b-collapse id="accordion-2" accordion="my-accordion" role="tabpanel">
        <b-card-body>
          <b-card-text>Comprehensive Security Solutions, with the most innovative Surveillance and Technology services, adapted to each client.</b-card-text>
        </b-card-body>
      </b-collapse>
    </b-card>
    <b-card no-body="" class="mb-3">
      <b-card-header header-tag="header" role="tab" class="p-1">
        <b-button block="" variant="warning">Cash</b-button>
      </b-card-header>
      <b-collapse id="accordion-3" accordion="my-accordion" role="tabpanel">
        <b-card-body>
          <b-card-text>Logistics, Cash, Management and Automation services for Financial Institutions, Banks and businesses.</b-card-text>
        </b-card-body>
      </b-collapse>
    </b-card>
  </div>
</div>
`;
